### PR TITLE
code style

### PR DESCRIPTION
**Pull request by haofanwang**
In parser.add_argument, it is redundant to keep both `default=None` and `required=False/True`.

### CODE REVIEW

**Review by sayakpaul — APPROVED** [review]
Appreciate your willingness to do this one!

File: examples/dreambooth/train_dreambooth_flax.py [hunk 1]
```diff
@@ -75,21 +73,19 @@ def parse_args():
...
         type=str,
         default=None,
-        required=False,
         help="A folder containing the training data of class images.",
     )
     parser.add_argument(
         "--instance_prompt",
         type=str,
-        default=None,
+        required=True,
```
Comment (patrickvonplaten) [inline examples/dreambooth/train_dreambooth_flax.py:88]: This is not the same thing no? Now we would get an error if we don't set `--instance_prompt` I think, but if someone doesn't want to do validation there is no need to set `--instance_prompt` I believe

Comment (patrickvonplaten) [inline examples/dreambooth/train_dreambooth_flax.py:88]:
```suggestion
```
We should remove this as well no?

**Comment by patrickvonplaten** [comment]
Hey @haofanwang,

Thanks a lot for the PR! I think `required=False` can always be omitted as well as `default=None` so thanks for cleaning this up! I think `required=True` statements should stay however

**Comment by haofanwang** [comment]
@patrickvonplaten In some args, `required=True` is not very important and there already exists a default value.

**Comment by haofanwang** [comment]
Sure. I will reformat later.

**Pull request closed without merging**